index.html

```diff
@@ -4,11 +4,13 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-    <link rel="stylesheet" href="stye.css" />
+    <link rel="stylesheet" href="style.css" />
     <title>Document</title>
   </head>
   <body>
-    <div class="json-section"></div>
+    <div class="container">
+      <div class="json-section"></div>
```

Comment:
`json-section` could be better named, maybe something like `to-do-list-container`, and then perhaps each item can be something like `to-do-list-item`